easybuild/easyconfigs/f/Flux/Flux-0.74.0-GCC-13.2.0.eb

```diff
@@ -0,0 +1,108 @@
+an entire cluster to themselves.
+"""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('make', '4.4.1'),
+    ('CMake', '3.27.6'),
+    ('jq', '1.7.1'),
+    ('Catch2', '3.8.1'),
+]
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('cffi', '1.15.1'),
+    ('PyYAML', '6.0.1'),
+    ('PLY', '3.11'),
+    ('Lua', '5.4.6'),
+    ('luaposix', '36.3'),
+    ('ZeroMQ', '4.3.5'),
+    ('Jansson', '2.14.1'),
+    ('hwloc', '2.9.2'),
+    ('lz4', '1.9.4'),
+    ('libedit', '20240517'),
+    ('Boost', '1.83.0'),
+    ('yaml-cpp', '0.8.0'),
+    ('PMIx', '4.2.6'),
+]
+
+components = [
+    ('flux-core', '0.74.0', {
+        'easyblock': 'ConfigureMake',
+        'source_urls': ['https://github.com/flux-framework/flux-core/releases/download/v%(version)s/'],
+        'sources': [SOURCELOWER_TAR_GZ],
+        'patches': [
+            'flux_core_0.74.0_test_tweak.patch',
+            'flux_core_0.74.0_shuffle_loop_for_gcc.patch',
+            'flux_core_0.74.0_reduce_test_parallelism.patch',
+        ],
+        'checksums': [
+            'de05fadf036e95a580eefe3329a092a37745ba5ac50657eb3f2ea00cd2e2db7e',
+            '3b91030fbfb0463ecc76140573d8c941899bbd191e08a8811daac909f997c18f',
+            '1a0b458ce191f3c11f90bf5c713f44a2d7c6f3b23165ac125fcd0065cd6cd489',
+            '84037c6c7ff9e025ee2a403dee1127acb2512b156ce408e88c64e6c6663c30ee',
+        ],
+        'start_dir': '%(namelower)s-%(version)s',
+        'runtest': 'check',
+    }),
+    ('flux-sched', '0.45.0', {
+        'easyblock': 'CMakeMake',
+        'source_urls': ['https://github.com/flux-framework/flux-sched/releases/download/v%(version)s/'],
+        'sources': [SOURCELOWER_TAR_GZ],
+        'checksums': ['2190858b687bf576c5e0ac409f678a67bc00c449a9f05e1f2b2284ae275a227f'],
+        'start_dir': '%(namelower)s-%(version)s',
+        'runtest': 'check',
+    }),
+    ('flux-pmix', '0.6.0', {
+        'easyblock': 'ConfigureMake',
+        'source_urls': ['https://github.com/flux-framework/flux-pmix/releases/download/v%(version)s/'],
+        'sources': [SOURCELOWER_TAR_GZ],
+        'checksums': [
+            '31f9d9b99f8a75c05e2fd183a35998f104d8e80bd819c57252aca477be0d4c6e',
+        ],
+        'start_dir': '%(namelower)s-%(version)s',
+        'runtest': 'check',
+    }),
+    ('flux-accounting', '0.46.0', {
+        'easyblock': 'ConfigureMake',
+        'source_urls': ['https://github.com/flux-framework/flux-accounting/releases/download/v%(version)s/'],
+        'sources': [SOURCELOWER_TAR_GZ],
+        'patches': [
+            'flux_accounting_0.46.0_fix_signed_unsigned_comparison.patch',
+        ],
+        'checksums': [
+            '552604291bee427d0e5b126045fcdaf49e8c9aac478c5022971e88b4b1447665',
+            '33221cc76a78511c46e42112089a75d1384b7bd75aa2de2e60f18632f269e9c2',
+        ],
+        'start_dir': '%(namelower)s-%(version)s',
+        'runtest': 'check',
```

Comment:
This particular test suite has a quite a few failing tests (200 out of ~1200), in the repo it is warned that it does not have a stable API. I'm inclined to switch off testing here as the module builds fine and is only a nice-to-have anyway.